GDJS/Runtime/runtimeobject.ts

```diff
@@ -219,6 +246,25 @@ namespace gdjs {
       initialInstanceData: InstanceData
     ): void {}
 
+    /**
+     * Update the effects during hot-reload
+     * @param oldEffectData Old effect data
+     * @param newEffectData New effect data
+     * @returns
+     */
+    _hotReloadRuntimeObjectEffects(
```

Comment:
This method is using only public methods (removeEffect, addEffect).
This means it could be done outside of the object, inside the hot reloader. So let's remove it from here and do it inside the hot reloader (because it will avoid cluttering gdjs.RuntimeObject, and avoid cluttering updateFromObjectData which would become ill named)

Reply:
Done 👍